### PR TITLE
CIAC-7639/Test-playbook-Local-Analysis-Alert-Investigation-for-xsiam

### PR DESCRIPTION
**Pull request by efelmandar**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-7639)

## Description
A new test playbook for XSIAM core playbook `Local Analysis Alert Investigation`

## Must have
- [x] Tests

**No mandatory dependencies were added on this pull request.**

### CODE REVIEW

File: Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml [hunk 1]
```diff
@@ -0,0 +1,561 @@
...
+    nexttasks:
+      '#default#':
+      - "16"
+      "yes":
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
```
Comment (OmriItzhak) [inline Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml:260]: Let's change it to 'is not empty'

File: Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml [hunk 2]
```diff
@@ -0,0 +1,561 @@
...
+    nexttasks:
+      '#default#':
+      - "16"
+      "yes":
+      - "18"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
```
Comment (OmriItzhak) [inline Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml:304]: Let's change it to 'is not empty'

File: Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml [hunk 3]
```diff
@@ -0,0 +1,561 @@
...
+          "y": 1400
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
```
Comment (OmriItzhak) [inline Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml:499]: Please add a check to verify if raising the incident severity to 'high' was successful.

File: Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml [hunk 4]
```diff
@@ -0,0 +1,561 @@
...
+  "18":
```
Comment (OmriItzhak) [inline Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml:500]: Please add a check to verify containment actions and include a task to unisolate the endpoint at the end of the playbook.

File: Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml [hunk 5]
```diff
@@ -0,0 +1,561 @@
...
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "16":
```
Comment (OmriItzhak) [inline Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml:473]: Please change the task to 'PrintErrorEntry' and provide a relevant message as the task's argument.

File: Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml [hunk 6]
```diff
@@ -0,0 +1,561 @@
+id: Local Analysis alert Investigation - Test
+version: -1
+name: Local Analysis alert Investigation - Test
+description: |-
+  This playbook tests the 'Local Analysis alert Investigation' playbook, which is part of the 'Core' pack. The playbook includes the following conducted tests:.
```
Comment (OmriItzhak) [inline Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml:5]: Please add a description to the playbook that outlines the specific tests it performs

File: Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml [hunk 7]
```diff
@@ -0,0 +1,934 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: 54c10b5e-0a7f-421e-84ce-9fe9af76c3c5
+    type: playbook
+    task:
+      id: 54c10b5e-0a7f-421e-84ce-9fe9af76c3c5
+      version: -1
+      name: Local Analysis alert Investigation
```
Comment (OmriItzhak) [inline Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml:51]: Replace it with the same playbook for updating the changes with the input Query.

File: Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml [hunk 8]
```diff
@@ -0,0 +1,934 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "19":
+    id: "19"
+    taskid: e653b5a3-f31c-4bfa-848c-f53fdde3faf7
+    type: playbook
+    task:
+      id: e653b5a3-f31c-4bfa-848c-f53fdde3faf7
+      version: -1
+      name: Local Analysis alert Investigation
```
Comment (OmriItzhak) [inline Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml:563]: Replace it with the same playbook for updating the changes with the input Query.

File: Tests/conf.json [hunk 9]
```diff
@@ -5565,6 +5575,9 @@
             "integrations": "Forcepoint Security Management Center",
             "playbookID": "ForcepointSecurityManagementCenter_test",
             "is_mockable": false
+        },
+        {
+            "playbookID": "0d7887b1-0266-407b-870a-f7b3e27fdccb"
```
Comment (ostolero) [inline Tests/conf.json:5580]: Wasn't able to find this ID, is this a new playbook? we usually have the identical id and name

**Review by OmriItzhak — APPROVED** [review]

**Pull request closed without merging**